requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+anymarkup==0.7.0
+flake8==3.5.0
+kubernetes~=12.0
+pylint~=2.6
+pytest~=6.2
+pytest-cov~=2.12
+responses
+testslide==2.6.3
```

Comment:
Feel free to make the testslide dependency of type `~=` too.

Reply:
All set.